src/components/views/game/game/GameView.scss

```diff
@@ -33,8 +33,8 @@
     position: absolute;
     left: 3%;
     bottom: 3%;
-    height: 50%;
-    width: 50%;
+    height: 340px;
+    width: calc(65vh);
```

Comment:
I would change this a bit higher (around 68vh-70vh)

Reply:
It's hard to see. I tested on different screens and 65vh would work, but 68vh has more headroom. :-)